db/cuckoo_table_db_test.cc

```diff
@@ -282,7 +282,7 @@ TEST_F(CuckooTableDBTest, SameKeyInsertedInTwoDifferentFilesAndCompacted) {
   }
 }
 
-TEST_F(CuckooTableDBTest, AdaptiveTable) {
+TEST_F(CuckooTableDBTest, DISABLED_AdaptiveTable) {
```

Comment:
Not sure whether we should keep supporting using DB with one type of table factory to open a previous db with sst files of another type.